base/applications/mspaint/toolsmodel.h

```diff
@@ -41,13 +70,18 @@ class ToolsModel
     int m_rubberRadius;
     BOOL m_transpBg;
     int m_zoom;
+    ToolBase *m_pTools[TOOL_MAX + 1];
```

Comment:
Why not make this a `CSimpleArray` or `CArray` ?

Reply:
Number of tools seems fixed.

Reply:
Number of tools seems fixed.

Reply:
The number of elements won't change. No need to use a dynamic array.

Reply:
This dynamic array also handles cleanup, which your static array does not do.